CorsixTH/Lua/hospital.lua

```diff
@@ -256,6 +249,18 @@ function Hospital:Hospital(world, avail_rooms, name)
   end
 end
 
+--! Checks if a room has been discovered
+--!param room_id (string) The name of the room
+--!return (boolean) true if found
+function Hospital:checkRoomDiscovery(room_id)
```

Comment:
I don't expect an answer with 'check', I expect that whatever you check ends up in a desired state.
When you check the oil in your car, you ensure that the oil level is (or becomes) good.

The common form for this kind of functions is to make it question-like, eg `isRoomDiscovered(room_id)`. It shows it's a query rather than an action, and it shows how to interpret the answer `true`, making it nicely readable in code like
``` lua
if hosp:isRoomDiscovered(room_id) then ...
```